test/support/channel_case.ex

```diff
@@ -25,7 +25,11 @@ defmodule Castle.ChannelCase do
     end
   end
 
-  setup _tags do
+  setup tags do
```

Comment:
This, like many other changes in this PR, is actually boilerplate when you init a phoenix project with Ecto.  (But Castle started out with Ecto, so it was missing).